### PR TITLE
Add the locationprefix option

### PR DESCRIPTION
**Pull request by erral**
This option allows to set a folder as a container for all backup and snapshot folders instead of overriding all location options.

It defaults to current `var` folder.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 3.0.1 (unreleased)
 ==================
 
+- Add a ``locationprefix`` option to configure a folder where all other
+  backup and snapstho folders will be created [erral]
```
Comment (gforcada) [inline CHANGES.rst]: typo

File: README.rst [hunk 2]
```diff
@@ -397,6 +397,10 @@ some system-wide directory like ``/var/zopebackups/instancename/`` and
     Location where backups are stored. Defaults to ``var/backups`` inside the
     buildout directory.
 
+``locationprefix``
+    Location of the folder where all other backup and snapshot folderis will
```
Comment (gforcada) [inline README.rst]: typo (missing space)

File: README.rst [hunk 3]
```diff
@@ -397,6 +397,10 @@ some system-wide directory like ``/var/zopebackups/instancename/`` and
     Location where backups are stored. Defaults to ``var/backups`` inside the
     buildout directory.
 
+``locationprefix``
```
Comment (gforcada) [inline README.rst]: probably @mauritsvanrees has more a saying here, but shouldn't it go with an underscore like all other options (I see a bit below ``only_blobs`` for example

**Comment by gforcada** [comment]
Thanks for that!! that will help make all our buildouts smaller, as with just one option would be enough :-)

I made some small cosmetic comments

**Comment by erral** [comment]
Thanks for the comments, I fixed them and squashed the commit.

**Comment by erral** [comment]
@mauritsvanrees done!

**Comment by mauritsvanrees** [comment]
Perfect, thank you.  I will merge.

**Comment by mauritsvanrees** [comment]
I have released 3.1 with this.

**Pull request merged**